spintest/__init__.py

```diff
@@ -25,16 +25,22 @@ def spintest(
     token: str = None,
     parallel: bool = False,
     verify: bool = True,
+    generate_report: str = None,
```

Comment:
```suggestion
    generate_report: Optional[str] = None,
```
(+ l'import de `Optional`)